SPECS/emu-manager.spec

```diff
@@ -13,6 +13,8 @@ BuildRequires:  json-c-devel
 BuildRequires:  libempserver-devel
 BuildRequires:  xcp-ng-generic-lib-devel
 
+Requires: %{name}-debuginfo = %{version}-%{release}
```

Comment:
Comment saying why

Reply:
Updated.

Reply:
Actually I meant why we need to require the debuginfo (for automated stacktrace output in case of crash), not why we need to require the exact version, which seems more obvious to me.

Reply:
Seems ok now.